spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -41,11 +37,17 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
  - <<mp-rest-client, MicroProfile Rest Client 2.0>>
 
+A MicroProfile 4.x compatible implementation must pass all the TCKs provided by the MicroProfile individual
+specifications. The Jakarta specifications TCKs are not required.
```

Comment:
I would say: The Jakarta specifications TCKs are not required. => The Jakarta specifications TCKs are optional.

Reply:
I'd remove the "4.x" verbiage. First, it does not apply to 4.0. Second, this spec applies to 4.1 by virtue of being in the 4.1 platform specification.  Recommended verbiage - "MicroProfile compatible implementations must pass ...".